code/modules/mob/living/carbon/human/human.dm

```diff
@@ -2015,3 +2015,21 @@ Eyes need to have significantly high darksight to shine unless the mob has the X
 			to_chat(usr, "[src] does not have any stored infomation!")
 	else
 		to_chat(usr, "OOC Metadata is not supported by this server!")
+
+/mob/living/carbon/human/verb/pose()
+	set name = "Set Pose"
+	set desc = "Sets a description which will be shown when someone examines you."
+	set category = "IC"
+
+	// no metagaming
+	if(!stat)
+		return
```

Comment:
Yup, intended

Reply:
To be clear, this will only allow them to set their pose if they're unconscious or dead. If that's the case, I question why 😄 but also, could you do `if(stat == CONSCIOUS)  => return` instead?